admin/con-rbac-config-permission-policies-external-file.adoc

```diff
@@ -1,50 +1,56 @@
+[id="con-rbac-config-permission-policies-external-file_{context}"]
 = Configuration of permission policies defined in an external file
 
-You can follow this approach of configuring the permission policies before starting the {product}. If permission policies are defined in an external file, then you can import the same file in the {product-short}. The permission policies need to be defined in Casbin rules format. For information about the Casbin rules format, see https://casbin.org/docs/category/the-basics[Basics of Casbin rules].
+You can follow this approach of configuring the permission policies before starting the {product}. If permission policies are defined in an external file, then you can import the same file in the {product-short}. The permission policies need to be defined in https://casbin.org/docs/category/the-basics[Casbin rules format].
 
 The following is an example of permission policies configuration:
 
-`p, role:default/guests, catalog-entity, read, deny`
-
-`p, role:default/guests, catalog.entity.create, create, deny`
-
-`g, user:default/<USER_TO_ROLE>, role:default/guests`
+[source]
+----
+p, role:default/guests, catalog-entity, read, deny
+p, role:default/guests, catalog.entity.create, create, deny
+g, user:default/<USER_TO_ROLE>, role:default/guests
+----
 
 If a defined permission does not contain an action associated with it, then add `use` as a policy. See the following example:
 
-`p, role:default/guests, kubernetes.proxy, use, deny`
+[source]
+----
+p, role:default/guests, kubernetes.proxy, use, deny
+----
 
 You can define the `policy.csv` file path in the `app-config.yaml` file:
 
 [source,yaml]
 ----
+# ...
 permission:
   enabled: true
   rbac:
     policies-csv-file: /some/path/rbac-policy.csv
+# ...
 ----
 
-== Mounting `policy.csv` file to the Developer Hub Helm Chart
+[id="con-rbac-config-permission-policies-external-file-mounting-policy_{context}"]
+== Mounting `policy.csv` file to the {helm}
 
-When the {product} is deployed with the Helm Chart, then you must define the `policy.csv` file by mounting it to the {product-short} Helm Chart. 
+If you are deploying {product} by using a Helm chart, you must define the `policy.csv` file by mounting it to the {helm}.
 
-You can add your `policy.csv` file to the {product-short} Helm Chart by creating a `configMap` and mounting it.
+You can add your `policy.csv` file to the {helm} by creating and mounting a config map.
 
 .Prerequisites
 
-* You are logged in to the OCP using the OpenShift web console.
-* {product} is installed and deployed using Helm Chart.
-+
-For more information about installing the {product} using Helm Chart, see xref:proc-install-rhdh-helm_admin-rhdh[]. 
+* You are logged in to {openshift}.
+* You have installed the {oc-cli}.
+* You have deployed {product} by using the xref:proc-install-rhdh-helm_admin-rhdh[{helm}].
 
 .Procedure
 
-. In Red Hat OpenShift, create a `configMap` to hold the policies as shown in the following example:
```

Comment:
Why did we remove the reference to RHOCP? I think we should continue to orient the user. But there might be an open PR that changes this to "From your OpenShift Container Platform account, create..." 

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/135 also adds attributes for OpenShift Container Platform and associated approved names.

Reply:
Because it is already mentioned in the prerequisites. Since there is already an open PR that covers some of this, I can just close this one since I only really opened it to get used to working on this doc set anyway.

I would check that any attributes being added are aligned with the [official product names list](https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1987148185) though, and I'd add the non-breaking spaces (`{nbsp}`) between words.

Reply:
The official product names list was used to develop the new attributes, but I welcome your eyes on the [updated attributes file ](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/135/files#diff-0f7e8a8fb3fdb835a98a8a937c0c87986260380b80b77a1f84f8ca529be430a2) since you're familiar with writing for that product. Btw, the line in the names list about using _Red Hat OpenShift_ in certain conditions is a bit unclear to me, so I did not add it to our attributes, but I'm sure you have a better understanding of the approved usage.